src/main/resources/locale_fr_FR.properties

```diff
@@ -192,7 +192,7 @@ command.setup.dialog.starting=Démarre l'installation.
 command.setup.dialog.welcome=Bienvenue dans la configuration du robot de réputation.
 command.thankwords.description=Configurer Thankwords
 command.thankwords.error.patternNotFound=Le modèle n'existe pas.
-command.thankwords.sub.add=Ajouter un autre motif. Les motifs sont interprétés comme des Regex. Un motif ne peut pas contenir d'espaces.
+command.thankwords.sub.add=Add a pattern. These are interpreted as regex. No spaces!
```

Comment:
English Translation in French

Reply:
Lel seems like I copied the wrong one xD